solr/core/src/java/org/apache/solr/core/NodeConfig.java

```diff
@@ -44,6 +44,8 @@
 
   private final Set<Path> allowPaths;
 
+  private final String allowUrls;
```

Comment:
I can't help but notice that just above here we have a `Set` of allowed paths, yet here you are adding a single String that is apparently more than one thing.  This is a bit out of balance.  If it's not much work, can this be harmonized?

Reply:
I'll see what it means to change the allowUrls to behave like allowPaths.
That means that users will be a little more verbose when listing the urls in xml, but it would be more consistent.

Reply:
Nevermind what I said. allowPaths is actually a single string in the xml file. It is parsed in SolrXmlConfig. I'll do the same for allowUrls.